packages/appstore/src/components/cfds-listing/cfds-listing.scss

```
@@ -675,6 +721,86 @@
     }
 }
 
+.dynamic-leverage-modal {
+    display: flex;
+    flex-direction: column;
+    gap: 2.4rem;
+    margin: 1.5rem 12.8rem 10rem;
+
+    @include mobile {
+        margin: 1.5rem 2.4rem 2.4rem;
+    }
+
+    position: absolute;
+    backface-visibility: hidden;
+    transition: transform 0.6s ease;
+    transform: rotateY(180deg);
+
+    &__content {
+        display: grid;
+        gap: 2.4rem;
+        grid-template-columns: 1fr 1fr;
+        margin: 0 10rem;
+
+        @include mobile {
+            grid-template-columns: 1fr;
+            margin: 0 6.5rem;
+        }
+    }
+
+    &__market {
+        border: 0.5px solid var(--border-hover);
+        border-radius: $BORDER_RADIUS * 2;
+        overflow: hidden;
+        background-color: var(--general-hover);
+        height: 24.8rem;
+
+        &-example {
+            font-style: italic;
+        }
+
+        &-table {
+            padding-bottom: 1rem;
+
+            &-header {
+                display: flex;
+                flex-direction: column;
+            }
+
+            &-row {
+                grid-template-columns: 1fr 0.5fr 1.25fr;
+                justify-items: center;
+                border-bottom: 0;
+                padding: 4px 0;
```

```suggestion
                padding: 0.4rem 0;
```
@arshad-rao-deriv nit-pick :)

---

Updated.